iis-wf/iis-wf-affmatching/src/test/java/eu/dnetlib/iis/wf/affmatching/match/voter/AffOrgMatchVoterStrengthEstimator.java

```diff
@@ -149,12 +140,25 @@ public static void classCleanup() throws IOException {
     @Test
     public void estimateVoterMatchStrengths_for_DocOrgRelationMatcher() throws IOException {
         
+        // given
+        
+        createInputDataFromJsonFiles(
+                ImmutableList.of("src/test/resources/experimentalData/input/all_organizations.json"),
```

Comment:
`src/test/resources/experimentalData/input/` base path appears in 42 places in all affmatch tests, we could consider extracting this root path to some shared test variable.